.github/workflows/linux.yml

```diff
@@ -29,3 +29,37 @@ jobs:
     - name: build
       run: |
         make -j 2
+        
+  linux-debug-parallel-simplex:
+    # simple parallel debug build using g++ with simplex configuration enabled
+
+    name: linux debug parallel simplex
+    runs-on: [ubuntu-18.04]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: modules
+      run: |
+        sudo apt-get install libopenmpi-dev numdiff
```

Comment:
```suggestion
        sudo apt-get install -y software-properties-common
        sudo add-apt-repository ppa:ginggs/deal.ii-9.2.0-backports
        sudo apt-get update
        sudo apt-get install -yq --no-install-recommends numdiff  libp4est-dev trilinos-all-dev petsc-dev
```

Reply:
Thanks @tjhei for loading also Trilinos and PETSc! I used the opportunity to enabled them during configuration! I also incorporated all your comments!